doc/user_doc/vic_admin/vicadmin_status_ref.md

```diff
@@ -0,0 +1,91 @@
+# Virtual Container Host Status Reference #
+
+The Web-based administration portal for virtual container hosts, VCH Admin, presents status information about a virtual container host.
+
+If the vSphere environment in which you are deploying a virtual container host does not meet the requirements, the deployment does not succeed. However, a successfully deployed virtual container host can stop functioning if the vSphere environment changes after the deployment. If environment changes adversely affect the virtual container host, the status of the affected component changes from green to yellow or red.
+
+## Virtual Container Host (VCH) ##
+
+VCH Admin checks the status of the processes that the virtual container host runs:
+
+- The port layer server, that ports Docker commands to vSphere API calls.
+- VIC Admin server, that 
+- The vSphere Integrated Containers Engine initialization service, that initializes virtual container hosts. 
+- The Docker engine server, that runs the vSphere Integrated Containers Engine implementations of Docker commands.
```

Comment:
Expanded, per your version on Slack, to "The Docker engine server, that exposes the Docker API and semantics, translating those composite operations into port layer primitives."